README

```diff
@@ -1 +1 @@
-README.rst
\ No newline at end of file
+README.rst
```

Comment:
Just curious, what is the change in this README, file? Github just makes it look like a whitespace change or something?

Reply:
Hmmm, good point. I didn't investigate that one any further since it just looked like a regular file on Windows, at the end of which a new line character is added. But now checking the satpy main branch on linux I realize this is in fact another symlink (to README.rst). And again, with the auto corrections injected by the pre-commit end-of-file-fixer on Windows, this symlink will become corrupted if checked out on linux afterwards:

 on Satpy main branch (on linux):
![image](https://github.com/user-attachments/assets/70270cd6-72aa-4034-9603-17c395727348)

on my contributing_and_rtd_build_fixes branch (on linux):
![image](https://github.com/user-attachments/assets/5949b08c-c796-40ff-b017-961d90cab663)

I also checked with cygwin on Windows, but the README file definitely appears as a regular file on that OS, here is the satpy main branch for instance:
![image](https://github.com/user-attachments/assets/4335b378-26b0-4ab2-95c1-07b990ade550)

So... there are a few ways to proceed from here:

1. We could again simply replace the README symlink with a **real** regular file containing a simple indication like: "Please refer to the README.rst file"
2. I have not tried that, but it might be possible to tweak the end-of-file-fixer hook to ignore some files, and in this case we could tell it to ignore the README "file"
3. I'm wondering why there is this README symlink since there is already the README.rst file next to it (as far as I know github would recognize any of README{.md,.rst,.txt} as a valid readme for a repository), so it might be a valid option to just remove the README element ?
4. You could forbid any development for satpy from Windows: this way there will be no issue due to the lack of support for symlinks... But that would be a bit drastic, would it not 😁?

Now the good thing is, I have just checked on the satpy main branch on linux and it seems those 2 files are really the only symlinks on the repository currently: 

```
kenshin@neptune:/mnt/data1/dev/projects/satpy$ find . -type l -ls
219043815      0 lrwxrwxrwx   1 kenshin  kenshin        10 août 28 20:20 ./README -> README.rst
219043787      0 lrwxrwxrwx   1 kenshin  kenshin        25 août 28 20:20 ./doc/source/dev_guide/CONTRIBUTING.rst -> ../../../CONTRIBUTING.rst
```

=> For now , I just removed the README element from the repository (so this is the option 3 above). Please let me know if you think we need to proceed differently on this point.